src/game_battler.cpp

```diff
@@ -455,7 +455,7 @@ int Game_Battler::ApplyConditions() {
 			}
 		}
 		else if(state.hp_change_type == state.ChangeType_gain) {
-			src_hp = std::min(source_hp, hp);
+			src_hp = hp;
 			if(src_hp < 0) {
```

Comment:
This healing number also appears to be able to exceed the rm2k3 damage cap of 999. Make a recovery skill of 100% + 999 for an actor with 9999 hp. It'll heal for 10998

Reply:
I checked it with both RPG_RT and the changes made in this PR: Your example indeed heals for 10998 while healing skills are capped to 9999 (tested it with a healing skill which heals 10002) in both RPG_RT and this PR. So everything works correctly.